submissions/semenjob/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <ul class="nav__list">
+        <li tabindex="1" class="header__list">
```

Comment:
Actually,  `tabindex`is not necessary for our task, WDYT?

Reply:
for the first time I encountered the implementation of moving through tabs .. the first option that I found. but as I understand it, it also works on links, so I think yes ... it's better to make all the items a link